### DISCUSSION TITLE
[Script request] Apache Guacamole

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#29] [Request script] [answered]
### Application Name

Apache Guacamole

### Website

https://guacamole.apache.org/

### Description

Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH.
We call it clientless because no plugins or client software are required.
Thanks to HTML5, once Guacamole is installed on a server, all you need to access your desktops is a web browser.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by IVRYSimon** [comment]
I was suprised it isnt in the directory yet. 
I really need a convenient install script. Building it without docker is always a hassle..

**Comment by trucke** [comment]
It would be nice to see this in the helper scripts.

At the moment I personally use the following (IMO, a very underrated project):
https://github.com/itiligent/Guacamole-Installer

Maybe you should have a look at this as a temporary solution. I've used it a couple times and never had a problem.

**Comment by quantumryuu** [comment]
Could take a look and try to create a script for it but not this week unfortunately

**Comment by michelroegl-brunner** [comment]
I am working on it in the moment. Should only be a few more days.

**Comment by MickLesk** [comment, marked as answer]
Added.
